automation/data_aggregator/builder.py

```diff
@@ -2,9 +2,57 @@
 
 
 def build_data_aggregator_class(manager_params):
-    output_format = manager_params["output_format"]
+    """
+    Suggest renaming data_directory and s3_bucket
+    to a consistent thing like "output_folder"
```

Comment:
How about `output_url` this way we handle any [pyFilesytem](https://www.pyfilesystem.org/) as long as it's installed.